src/main/java/macrobase/analysis/summary/result/DatumWithScore.java

```diff
@@ -2,6 +2,7 @@
 
 import macrobase.datamodel.Datum;
 
+// TODO: REMOVE
```

Comment:
we've got a problem that contextual Outlier Detection is still using this.. So we'll have to keep it as a TODO for now.